### PR TITLE
update comments

### PR DESCRIPTION
**Pull request by themiswang**
#no-changelog

### CODE REVIEW

**Review by samedson — APPROVED** [review]
🚢

File: Crashlytics/Crashlytics/Components/FIRCLSApplication.m [hunk 1]
```diff
@@ -47,9 +49,9 @@
 #elif TARGET_OS_TV
   return @"tvos";
 #elif TARGET_OS_WATCH
-  return @"ios";  // TODO: temporarily use iOS until Firebase can add watchos to the backend
+  return @"ios";
 #elif defined(TARGET_OS_VISION) && TARGET_OS_VISION
-  return @"ios";  // TODO: temporarily use iOS until Firebase can add visionos to the backend
```
Comment (andrewheard) [inline Crashlytics/Crashlytics/Components/FIRCLSApplication.m:52]: Were these TODOs intentionally removed?

Comment (themiswang) [reply Crashlytics/Crashlytics/Components/FIRCLSApplication.m:52]: Yes, Follow back on https://github.com/firebase/firebase-ios-sdk/pull/11938#discussion_r1358798948, indeed it is a legacy function, Crashlytics should have proper support for watchOS, but we need to followup on backend change for VisionOS with team later.

Comment (themiswang) [reply Crashlytics/Crashlytics/Components/FIRCLSApplication.m:52]: The TODOs are not valid any more.

**Pull request merged**